docs/ContribOperators.md

```diff
@@ -2135,6 +2136,81 @@ This version of the operator has been available since version 1 of the 'com.micr
 </dl>
 
 
+### <a name="com.microsoft.GemmFloat8"></a><a name="com.microsoft.gemmfloat8">**com.microsoft.GemmFloat8**</a>
+
+  Gemm for float and float 8. The operator calls function 'cublasLtMatmul'
```

Comment:
Could you move the cuda requirement to a seperate paragraph? I'd like to add the ROCm support for it (for a very narrow inference case, tho).